src/main/java/io/stargate/sgv2/jsonapi/api/v1/NamespaceResource.java

```diff
@@ -102,4 +128,30 @@ public Uni<RestResponse<CommandResult>> postCommand(
         // map to 2xx unless overridden by error
         .map(commandResult -> commandResult.map());
   }
```

Comment:
Looks like the only difference for these two method keyspaceCommand and namespaceCommand is 
how to construct the schemaObject?

Possible to exact that out?

Reply:
I see.

Reply:
@Yuqi-Du As I make changes for new commands I don't changes in these method, so made it back to single method as suggested.